### DISCUSSION TITLE
Another macOS install problem

### DISCUSSION THREAD

**Discussion by dwsounds333** [publicsamples/Quetzalcoatl#18] [General]
I'm stuck in an infinite loop of installing the file to extract the samples and then selecting the sample folder and being told it was installed successfully and to re-launch the vst. also wondering is there an easier way to install with the paid version? Any help would be greatly appreciated. Thank you

**Comment by RickTaylor79** [comment]
That's weird, so after you relaunch the plugin it asks you to reinstall the samples? 

Can you send a screenshot of the Library/Application Support/modularsamples/Quetzalcoatl folder? Assuming it exists.. 

There's a pre-release update here:

https://github.com/publicsamples/Quetzalcoatl/releases/tag/0.6.6

You might try this and see if it makes any difference..

**Comment by RickTaylor79** [comment]
Also, if I could get your OS version and system specs that might help.. 

The paid version is identical, except for additional samples and presets.

**Comment by dwsounds333** [comment]
that folder doesn't exist. I'm using Ventura 13.0.1 on 2018 Mac Mini. Running ableton live 12.0.15. Downloaded the newest update and am further confused as where to even start with installation. Is there a step by step install guide somewhere that I may have missed?

**Reply by RickTaylor79** [reply]
Okay, I fired up another Mac and got exactly the same thing you've been experiencing. It works fine with the laptop I built it on, but nothing else it seems. Not sure why this is happening, but here's a fix.

1. Download these two multipart zips. [One](https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.6/Quetzalcoatl.Mac.VTS3.zip.001) & [Two](https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.6/Quetzalcoatl.Mac.VTS3.zip.002).
2. Extract them with [the Unarchiver](https://theunarchiver.com)
3. Open the install package by **right clicking** and selecting open. A warning will pop that looks like this:

<img width="262" alt="Screenshot 2024-08-02 at 10 27 05" src="https://github.com/user-attachments/assets/c7f8e1c5-98e7-4d73-b309-4c6fbf37f692">


4. Click "Open to proceed" & complete the installation 

That's it, you should now have a working copy of Quetzalcoatl.
